### PR TITLE
[WDFT-AMS-102020] Yann Quintin

### PR DESCRIPTION
**Pull request by YannQuintin**
Submitting first iteration with layout, styling but missing the main font.

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -6,3 +6,189 @@ Green: #00B172
...
+  }
+
+  body {
+    font-family: "Sans Serif";
+    font-size: 10px;
+    line-height: 3.5em;
+    font-weight: 300;
+  }
+
+/* |||||||| Navbar |||||||| */
```
Comment (wax-bear) [inline styles/style.css:30]: Is this comment necessary? The same question goes for the other comments.

File: styles/style.css [hunk 2]
```diff
@@ -6,3 +6,189 @@ Green: #00B172
 White: #FFF
 
 */
+
+html, body {
+    font: "Sans Serif";
+    margin: 0;
+    padding: 0;
+  }
+
```
Comment (wax-bear) [inline styles/style.css:15]: Please mind the indentation and blank lines.

File: index.html [hunk 3]
```diff
@@ -1,18 +1,63 @@
...
+    <link rel="stylesheet" href="/styles/style.css">
   </head>
   <body>
-    Premium Discover Help Download Music for everyone. Spotify is now free on mobile, tablet and computer. Listen to the
-    right music, wherever you are. What’s on Spotify? Millions of Songs There are millions of songs on Spotify HD Music
-    Listen to music as if you were listening live Stream Everywhere Stream music on your smartphone, tablet or computer
-    It’s as yeezy as Kanye West. Search Know what you want to listen to? Just search and hit play. Browse Check out the
-    latest charts, brand new releases and great playlists for right now. Discover Enjoy new music every Monday with your
-    own personal playlist. Or sit back and enjoy Radio.
+    <Navbar class="nav-bar">
```
Comment (wax-bear) [inline index.html:13]: What is the "Navbar" element? Is this HTML, or something else?

File: styles/style.css [hunk 4]
```diff
@@ -6,3 +6,189 @@ Green: #00B172
...
+    }
+
+    .app-functionalities-text {
+        width: 380px;
+        font-size: 2.3em;
+        text-align: left;
+    }
+
+
+    section:nth-of-type(2n) {
```
Comment (wax-bear) [inline styles/style.css:180]: Could you please explain the use of using ":nth-of-type(2n)" here?

**Review by wax-bear — COMMENTED** [review]
Great job Yann!
The website looks good.
Please checkout my comments for potential improvements.

File: styles/style.css (hunk 1 above)
Comment (YannQuintin) [reply styles/style.css:30]: Not necessary, no.

I am using this type of  comment to differentiate the  sections. It's a visual help.

File: index.html (hunk 3 above)
Comment (YannQuintin) [reply index.html:13]: I wrote this thinking it was the HTML naming convention. I'll change it to the correct <nav></nav>.

File: styles/style.css (hunk 4 above)
Comment (YannQuintin) [reply styles/style.css:180]: Two reasons:
1. I was struggling with the placement of the logo in the background. I was not able to nest it elsewhere ( the section of in its own container) and make the logo placement work.
2. Had not done it yet and  knew it was possible to do so.  

bottom line, where do you recommend I place it?

Comment (wax-bear) [reply styles/style.css:180]: Hey Yann,

Apologies for my late reply.
What about applying an id to the section, with the style declarations in it?
Imagine, a future you or another developer will add a section _above_ the section you are targeting. Do you see the potential risk of doing that?

Please reach out if you have a question. :)

**Pull request closed without merging**